config/betamocks/services_config.yml

```diff
@@ -363,14 +363,6 @@
       :uid_location: header
       :uid_locator: 'va_eauth_pnid'
 
-  # PPIU payment information
-  - :method: :get
-    :path: "/wss-ppiu-services-web/rest/ppiuServices/v1/paymentInformation"
-    :file_path: "evss/ppiu/payment_information"
-  - :method: :post
-    :path: "/wss-ppiu-services-web/rest/ppiuServices/v1/paymentInformation"
-    :file_path: "evss/ppiu/update_payment_information"
```

Comment:
I'm removing PPIU from mockdata repo in https://github.com/department-of-veterans-affairs/vets-api-mockdata/pull/618